KIF Tests/SystemTests.m

```diff
@@ -33,6 +33,14 @@ @interface SystemTests : SenTestCase
 
 @implementation SystemTests
 
+- (void)setUp {
+    [[NSNotificationCenter defaultCenter] postNotificationName:@"SystemTestsNotification" object:nil userInfo:@{@"setup" : @YES}];
+}
+
+- (void)tearDown {
+    [[NSNotificationCenter defaultCenter] postNotificationName:@"SystemTestsNotification" object:nil userInfo:@{@"setup" : @NO}];
```

Comment:
Ah, that was just being a little lazy when I tried to cram everything into the system tests controller. That code isn't necessary when I revert to the older solution.